docs/gce-quickstart.md

```diff
@@ -0,0 +1,134 @@
+# Quickstart for Using the Oracle Toolkit for GCP on GCE
+
+This document serves as a quickstart guide and simple reference for using the Oracle Toolkit for GCP on Google Compute Enging (GCE). It is an abridged version of the more comprehensive [Oracle Toolkit for GCP - GCE User Guide](gce-user-guide.md).
+
+## Prerequisite Assumptions
+
+This document assumes that you have:
+
+1. Provisioned your Ansible Control Node, installed Ansible and JMESpath on it, and downloaded the toolkit.
+1. Setup Google Cloud foundational components such as IAM, networking, GSC storage buckets with the required media staged, and security aspects.
+1. Have either Terraform or the Google Cloud CLI and specifically the **gcloud** utility installed.
+
+If you need additional details to setup any of these prerequisites, refer to Google documentation such as [Google Cloud quickstarts and tutorials](https://cloud.google.com/docs/tutorials) and specifically [Install the gcloud CLI](https://cloud.google.com/sdk/docs/install).
+
+## Deploying using the Google Cloud CLI
+
+### Set Supporting Variables
+
+Before beginning, infrastructure locality and networking aspects must be defined and captured (i.e. into shell environment variables for convenience):
+
+```bash
+PROJECT_ID=PROJECT_ID
+REGION_ID=REGION
+ZONE_ID=ZONE
+NETWORK_ID=NETWORK
+SUBNET_ID=SUBNET
+
+gcloud config set project ${PROJECT_ID}
+```
+
+### Create the GCE Instance and Block Storage Devices
+
+Specify some instance specific characteristics such as the VM shape, name, and OS image:
+
+```bash
+MACHINE_TYPE="c4-standard-4"
+IMAGE_FILE="$(gcloud compute images describe-from-family rhel-8 --project=rhel-cloud --format json | jq -r '.selfLink')"
+VM_NAME=INSTANCE_NAME
+```
+
+> **NOTE:** Some operating systems such as Red Hat Enterprise Linux may have additional licensing costs. See the [Premium images](https://cloud.google.com/compute/disks-image-pricing?hl=en#section-1) section of Google documentation for additional details.
+
+Create the instance (add network tags by appending the `--tags TAG` option as required):
+
+```bash
+gcloud compute instances create ${VM_NAME} \
+  --project=${PROJECT_ID} \
+  --zone=${ZONE_ID} \
+  --machine-type=${MACHINE_TYPE} \
+  --network-interface=network-tier=STANDARD,stack-type=IPV4_ONLY,subnet=${SUBNET_ID} \
+  --create-disk=auto-delete=yes,boot=yes,device-name=${VM_NAME}-boot-disk,image=${IMAGE_FILE},mode=rw,provisioned-iops=3300,provisioned-throughput=290,size=64G,type=hyperdisk-balanced
```

Comment:
Just to reduce commands, would it make sense to create the other disks here too?